src/components/Sponsors.jsx

```diff
@@ -1,31 +1,43 @@
 import Image from 'next/image'
+import { useTheme } from 'next-themes'
 
 import { Container } from '@/components/Container'
 import sponsors from '@/data/sponsors.json'
 
 export function Sponsors() {
+  const { theme } = useTheme()
+
   return (
     <section
       id="sponsors"
       aria-label="Sponsors"
-      className="bg-jordi-black py-20 sm:py-32"
+      className="bg-jordi-beige-100 py-20 dark:bg-black sm:py-32"
     >
       <Container>
-        <h2 className="mx-auto max-w-2xl text-center font-display text-4xl font-medium tracking-tighter text-jordi-orange-600 sm:text-5xl">
+        <h2 className="mx-auto max-w-2xl text-center font-display text-4xl font-medium tracking-tighter text-jordi-orange-500 sm:text-5xl">
           Sponsorships
```

Comment:
I don't know if you prefer ```Exclusive Sponsorship``` or only ```Sponsorship``` as it is.

Reply:
Exclusive Sponsorship, sounds better